src/module/item/consumable/spell-consumables.ts

```diff
@@ -31,26 +36,50 @@ const wandCompendiumIds: Record<number, string | undefined> = {
     9: "Fgv722039TVM5JTc",
 };
 
-function getIdForSpellConsumable(type: "scroll" | "wand", heightenedLevel: number): string | null {
-    return type === "scroll"
-        ? scrollCompendiumIds[heightenedLevel] ?? null
-        : wandCompendiumIds[heightenedLevel] ?? null;
+function getIdForSpellConsumable(type: SpellConsumableItemType, heightenedLevel: number): string | null {
+    let id = null;
+    switch (type) {
+        case "cantrip-deck-5":
+            id = cantripDeckId;
+            break;
+        case "scroll":
+            id = scrollCompendiumIds[heightenedLevel] ?? null;
+            break;
+        default:
+            id = wandCompendiumIds[heightenedLevel] ?? null;
+    }
+    return id;
 }
 
-function getNameForSpellConsumable(type: "scroll" | "wand", spellName: string, heightenedLevel: number): string {
-    if (type === "scroll") {
-        return game.i18n.format("PF2E.ScrollFromSpell", { name: spellName, level: heightenedLevel });
-    } else {
-        return game.i18n.format("PF2E.WandFromSpell", { name: spellName, level: heightenedLevel });
+function getNameForSpellConsumable(type: SpellConsumableItemType, spellName: string, heightenedLevel: number): string {
+    let name;
+    switch (type) {
+        case "cantrip-deck-5":
+            name = game.i18n.format("PF2E.CantripDeckFromSpell", { name: spellName });
+            break;
+        case "scroll":
+            name = game.i18n.format("PF2E.ScrollFromSpell", { name: spellName, level: heightenedLevel });
+            break;
+        default:
```

Comment:
Don't use default for this. Use "wand". That way if anything is ever added, typescript will notify you that a change is required.

Reply:
Even better, have a mapping from item type to localization entry. Then pull from that mapping.